relm4-macros/src/widgets/gen/util/widget.rs

```diff
@@ -18,11 +18,13 @@ impl ViewWidgets {
 
 impl Widget {
     // Find all template children and get their variables in scope.
-    pub(crate) fn get_template_child_in_scope(&self, stream: &mut TokenStream2) {
+    pub(crate) fn get_template_child_in_scope(&self, stream: &mut TokenStream2, init: bool) {
```

Comment:
I'd probably prefer an enum over a boolean for this, it's unclear what's going on from the call-site.

Reply:
I've added an enum now :)